content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -247,20 +262,85 @@ zones, or regions.
     caption="A service mesh after a control plane instance fails"
     >}}
 
-The following list ranks control plane deployment examples by availability:
+### Independent Control Planes
```

Comment:
This term has not been defined or agreed upon. We probably shouldn't use it. In the old terminology, this would have effectively been "replicated". We had previously considered using the term `independent control planes` to represent the case where both are primaries and there is no cross-cluster load balancing between the clusters

Reply:
I do like independent control planes way better than replicated control planes :)

Reply:
The problem is that it's not defined and we haven't reached agreement on the term. There doesn't really seem to be a need for a new term. We could just as easily call this section `Control plane per cluster`.